TipCatDotNet.Api/Services/HospitalityFacilities/MemberServiceExtensions.cs

```diff
@@ -35,6 +35,40 @@ public static Result EnsureCurrentMemberBelongsToAccount(this Result result, int
         }
 
 
+        public static async Task<Result> EnsureTargetAccountHasNoDefault(this Result result, AetherDbContext context, int? targetAccountId,
```

Comment:
Should be moved to another class, because relates to an account, but not to a member

Reply:
Can I rename MemberServiceExtension to ModelEnsuresExtention/ServiceEnsuresExtention. Because i already have a lot of checkings with facilities and accounts in this extenstion. If I moved all of this to separate extentions, I would create files. And it litter our project. Or should i think about separate file by directories(f.e. Extentions, Validators, Requests, Responses)